etl/steps/data/garden/ember/2022-12-13/yearly_electricity.py

```diff
@@ -0,0 +1,387 @@
+"""Garden step for Ember's Yearly Electricity Data.
+
+"""
+
+import pandas as pd
+from owid import catalog
+from owid.datautils import geo
+from shared import CURRENT_DIR, add_population, add_region_aggregates, log
+
+from etl.paths import DATA_DIR
+
+# Details for dataset to export.
+DATASET_SHORT_NAME = "yearly_electricity"
+# Details for dataset to import.
+MEADOW_VERSION = "2022-12-13"
+MEADOW_DATASET_PATH = DATA_DIR / f"meadow/ember/{MEADOW_VERSION}/{DATASET_SHORT_NAME}"
+
+COUNTRY_MAPPING_PATH = CURRENT_DIR / f"{DATASET_SHORT_NAME}.countries.json"
+METADATA_PATH = CURRENT_DIR / f"{DATASET_SHORT_NAME}.meta.yml"
```

Comment:
I believe that you could use:

```python
from etl.helpers import Names
N = Names(__file__)

# Details for dataset to export.
DATASET_SHORT_NAME = N.short_name
...

COUNTRY_MAPPING_PATH = N.country_mapping_path
METADATA_PATH = N.metadata_path
```

Also, if the meadow dataset has the same version, you can simply use the following to load it:

```python
ds_meadow = N.meadow_dataset
```

Reply:
You are right, but I think I'm not going to make these style changes now, to avoid blocking this update.